homeassistant/components/modbus/sensor.py

```diff
@@ -47,6 +47,7 @@ def __init__(
     ) -> None:
         """Initialize the modbus register sensor."""
         super().__init__(hub, entry)
+        self._attr_state_class = entry.get(CONF_STATE_CLASS)
```

Comment:
It would be nice to have a default and not just None

Reply:
In KNX it shouldn't https://github.com/home-assistant/core/pull/53996#discussion_r683016047
I just did it same here, but Modbus may be different as it doesn't allow string values afaik. What would be the right state_class then, `measurement`?

Reply:
Modbus allows string values! Leave it as it is.